plugins/DaphneController.cpp

```diff
@@ -15,39 +15,612 @@
 
 #include <string>
 #include <logging/Logging.hpp>
+#include <fstream>
+#include <iomanip>
+#include <ctime>
+#include <regex>
+#include <stdexcept>
+#include <cmath>
+#include <chrono>
+#include <bitset>
+#include <thread>
+#include <algorithm>
 
 namespace dunedaq::daphnemodules {
 
 DaphneController::DaphneController(const std::string& name)
   : dunedaq::appfwk::DAQModule(name)
 {
   register_command("conf", &DaphneController::do_conf);
+  register_command("dump_buffers", &DaphneController::dump_buffers);
 }
 
-void
-DaphneController::init(const data_t& /* structured args */)
-{
-  TLOG() << "Test";
-}
 
 void
 DaphneController::get_info(opmonlib::InfoCollector& ci, int /* level */)
 {
-  daphnecontrollerinfo::Info info;
-  info.total_amount = m_total_amount;
-  info.amount_since_last_get_info_call = m_amount_since_last_get_info_call.exchange(0);
 
-  ci.add(info);
+  static const std::regex volt_regex(".* VBIAS0= ([^ ]+) VBIAS1= ([^ ]+) VBIAS2= ([^ ]+) VBIAS3= ([^ ]+) VBIAS4= ([^ ]+) POWER.-5v.= ([^ ]+) POWER..2.5v.= ([^ ]+) POWER..CE.= ([^ ]+) TEMP.Celsius.= ([^ ]+) .*");
```

Comment:
Yes, that is a problem. At the moment with only one access to the board (this code) slow control won't be able to access the board. 
From what Manuel said, this is only for the current version of the daphne. Next version will allow this. 
So that's what we do for now. 

I'll check with Manuel, but I think the metrics you suggested are not available from this interface. THey should come from the readout. But I need to investigate, as this is out of my control.